coderd/httpmw/apikey_test.go

```diff
@@ -435,7 +436,7 @@ func TestAPIKey(t *testing.T) {
 		gotAPIKey, err := db.GetAPIKeyByID(r.Context(), id)
 		require.NoError(t, err)
 
-		require.NotEqual(t, sentAPIKey.IPAddress, gotAPIKey.IPAddress)
+		require.Equal(t, net.ParseIP("1.1.1.1"), gotAPIKey.IPAddress.IPNet.IP)
```

Comment:
noob question: I see you added `:3555` on L417 but then here on L439, you're passing in `"1.1.1.1`. So why add the `:3555` if we're not testing it here? (or are we??)

Reply:
We're testing that it gets stripped correctly!

Reply:
ahh! I misread - that makes sense! thanks Colin!